backend/utils/gridUtils.js

```diff
@@ -0,0 +1,10 @@
+export function extractOrderedUids (grid) {
+  try {
+    return Object.keys(grid.cols)
+      .map(key => grid.cols[key])
+      .sort((a, b) => a.order - b.order)
```

Comment:
what's the content of `order`?

Reply:
I believe it's a zero based index representing the where the column lies from left to right